core/Tracker/VisitExcluded.php

```diff
@@ -180,13 +180,26 @@ private function isIpInRange()
         if ($cache->contains($key)) {
             $isInRanges = $cache->fetch($key);
         } else {
-            $isInRanges = $ip->isInRanges($this->getBotIpRanges());
+            if ($this->isChromeDataSaverUsed($ip)) {
+                $isInRanges = false;
+            } else {
+                $isInRanges = $ip->isInRanges($this->getBotIpRanges());
+            }
+
             $cache->save($key, $isInRanges);
         }
 
         return $isInRanges;
     }
 
+    private function isChromeDataSaverUsed(IP $ip)
+    {
+        // see https://github.com/piwik/piwik/issues/7733
+        return !empty($_SERVER['HTTP_VIA'])
+            && false !== strpos(strtolower($_SERVER['HTTP_VIA']), 'chrome-compression-proxy')
+            && $ip->isInRange('66.249.80.0/20');
```

Comment:
> I don't know all googles IP ranges. Are they specified somewhere? 

they are here, need to be refactored out: https://github.com/piwik/piwik/blob/2.14.3/core/Tracker/VisitExcluded.php#L193-L203